src/tools/user-agent-parser/user-agent-result-cards.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div>
+    <n-grid :x-gap="12" :y-gap="8" cols="1 s:2" responsive="screen">
+      <n-gi v-for="{ heading, icon, content } in sections" :key="heading">
+        <n-card style="height: 100%">
+          <n-page-header>
+            <template #title>
+              {{ heading }}
+            </template>
+            <template v-if="icon" #avatar>
+              <n-icon size="30" :component="icon" :depth="3" />
+            </template>
+          </n-page-header>
+
+          <br />
+
+          <n-space>
+            <span v-for="{ label, getValue } in content" :key="label">
+              <n-tooltip trigger="hover">
+                <template #trigger>
+                  <n-tag v-if="getValue(userAgentInfo)" type="success" size="large" round :bordered="false">
+                    {{ getValue(userAgentInfo) }}
+                  </n-tag>
+                  <span v-else />
+                </template>
+                {{ label }}
+              </n-tooltip>
```

Comment:
Is their something preventing from putting the v-if on the n-tooltip to avoid the empty span ?

```suggestion
              <n-tooltip trigger="hover" v-if="getValue(userAgentInfo)">
                <template #trigger>
                  <n-tag type="success" size="large" round :bordered="false">
                    {{ getValue(userAgentInfo) }}
                  </n-tag>
                </template>
                {{ label }}
              </n-tooltip>
```

Reply:
Jesus. I could have probably thought of that myself.